modules/EnsEMBL/Web/Component/Variation/Summary.pm

```diff
@@ -226,7 +226,7 @@ sub variation_source {
     } 
   } elsif ($source =~ /ClinVar/i) {
     $sname = ($name =~ /^rs/) ?  'CLINVAR_DBSNP' : 'CLINVAR';
-    $source_link = $hub->get_ExtURL_link("About $source", $sname, $name);
+    $source_link = $hub->get_ExtURL_link("View in $source", $sname, $name);
```

Comment:
The change should be made only if the variant name is numerical, so we should add a conditional - 

```suggestion
    $source_prefix ($name =~ /^rs/) ?  'About' : 'View in';
    $source_link = $hub->get_ExtURL_link("$source_prefix $source", $sname, $name);
```